src/internal/index.ts

```diff
@@ -171,6 +176,9 @@ export class RTMAPI {
 }
 
 export interface APIError {
-  type: string;
-  message: string;
+  error: {
+    type: string;
+    message: string;
+    data?: unknown;
+  };
```

Comment:
It is breaking change, not sure if we can introduce it now, wdyt?

Reply:
You're right, this is breaking change and I don't think it's worth making it.